.github/workflows/translations-changeset.yml

```diff
@@ -0,0 +1,41 @@
+name: Create translations patch
+
+on:
+  pull_request:
+    types:
+      - opened
+    branches:
+      - main
+
+jobs:
+  create-translations-patch:
+    if: github.actor == 'bc-svc-local'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+
+      - name: Create a translations changeset
+        run: |
+          mkdir -p .changeset
+          echo "---
+          \"@bigcommerce/catalyst-core\": patch
+          ---
+
+          Update translations." > .changeset/translations-patch.md
+
+      - name: Commit changeset
+        run: |
+          git config --global user.name 'bc-svc-local'
+          git config --global user.email 'bc-svc-local@users.noreply.github.com'
+          git add .changeset/translations-patch.md
```

Comment:
What if we don't release for a couple weeks and there are multiple translations PRs? Is there any way we can add a rng postfix to the file name so we don't run into any file conflicts?

Reply:
I can see the benefit now of having multiple changesets, in the change log when it links to the commit. I will look into it.